src/Directory.Build.props

```diff
@@ -9,4 +9,16 @@
     <ShouldUnsetParentConfigurationAndPlatform>false</ShouldUnsetParentConfigurationAndPlatform>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetOS>linux</TargetOS>
+    <TargetOS Condition="$([MSBuild]::IsOSPlatform('OSX'))">osx</TargetOS>
+    <TargetOS Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">windows</TargetOS>
```

Comment:
What if we build package on windows machine, but use it on mac? After all, initially it's just msil.

Reply:
The native build happens on the SDK, and then they are cross targeting, the SDK will need all profiles to do the right thing.  I'm pretty sure this is not desirable.  There is likely something that may need doing for sourcebuild, I doubt they have access to all of the packages.

This targetOS will however likely be useful when we enable optimization for protobuild.

Reply:
The native build happens on the SDK, and then they are cross targeting, the SDK will need all profiles to do the right thing.  I'm pretty sure this is not desirable. Just delete this whole property group.